cmd/cephcsi.go

```diff
@@ -282,6 +265,28 @@ func main() {
 	os.Exit(0)
 }
 
+func setPIDLimit(conf *util.Config) {
+	// set pidLimit only for NodeServer
+	// the driver may need a higher PID limit for handling all concurrent requests
+	if conf.IsNodeServer && conf.PidLimit != 0 {
+		currentLimit, pidErr := util.GetPIDLimit()
```

Comment:
```suggestion
		_, pidErr := util.GetPIDLimit()
```
`currentLimit` variable is not used anywhere, let's remove it.

Reply:
its used for logging.